src/Client.php

```diff
@@ -75,7 +75,8 @@ public function post($url, $organizationId, array $data = [], array $params = []
         return $this->processResult($this->httpClient->post(
             $url,
             [
-                'query' => $this->getParams($organizationId, $data) + $params,
+                'query' => $params + $this->getParams($organizationId),
+                'form_params' => ['JSONString' => json_encode($data)],
```

Comment:
Because `getParams()` encodes a JSON to be sent in a query string, and here we are encoding the data to be sent inside the request body.